### DISCUSSION TITLE
RancherDesktop (v 0.6.1) for Windows stuck on "Waiting for Kubernetes API"

### DISCUSSION THREAD

**Discussion by lamski** [rancher-sandbox/rancher-desktop#1054] [Q&A]
Hi,

I installed RancherDesktop (version 0.6.1) on my Windows 10 machine:
```
OS Name:                   Microsoft Windows 10 Enterprise
OS Version:                10.0.19042 N/A Build 19042
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Member Workstation
OS Build Type:             Multiprocessor Free

System Manufacturer:       Dell Inc.
System Model:              Precision 5520
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 158 Stepping 9 GenuineIntel ~2901 Mhz
Total Physical Memory:     16,232 MB
```
The application installs successfully, but it gets stuck on "Waiting for Kubernetes API"

![image](https://user-images.githubusercontent.com/24830708/144508230-8c1baa41-5ba9-4c4b-bf61-ae986ac500db.png)

Am I missing some installatin steps?

**Comment by evertonlperes** [comment]
> Hi,
> 
> I installed RancherDesktop (version 0.6.1) on my Windows 10 machine:
> 
> ```
> OS Name:                   Microsoft Windows 10 Enterprise
> OS Version:                10.0.19042 N/A Build 19042
> OS Manufacturer:           Microsoft Corporation
> OS Configuration:          Member Workstation
> OS Build Type:             Multiprocessor Free
> 
> System Manufacturer:       Dell Inc.
> System Model:              Precision 5520
> System Type:               x64-based PC
> Processor(s):              1 Processor(s) Installed.
>                            [01]: Intel64 Family 6 Model 158 Stepping 9 GenuineIntel ~2901 Mhz
> Total Physical Memory:     16,232 MB
> ```
> 
> The application installs successfully, but it gets stuck on "Waiting for Kubernetes API"
> 
> ![image](https://user-images.githubusercontent.com/24830708/144508230-8c1baa41-5ba9-4c4b-bf61-ae986ac500db.png)
> 
> Am I missing some installatin steps?

Hi @lamski 
Are you using any VPN application or Security software along with RD?

Also, could you try the latest version `0.7.1`? https://github.com/rancher-sandbox/rancher-desktop/releases/tag/v0.7.1

**Comment by loicmathieu** [comment]
Same here.

I had to disabled my VPN in order Rancher Desktop to start properly.
I also had to start docker manually inside WSL for this to work but this may be due to my WSL having docker installed before installing Rancher Desktop.

EDIT: the following was a false alarm due to my docker runtime not started
~But even after that, a simple `docker ps` or `docker images` leads to an error so I'm not sure it's properly installed.~

**Comment by evertonlperes** [comment]
Related to the VPN issues, we're working on this epic (https://github.com/rancher-sandbox/rancher-desktop/issues/1115) in order to fix the VPN issues.
Also I'd recommend to upgrade your Rancher Desktop version to `1.0.0-beta1` -> https://github.com/rancher-sandbox/rancher-desktop/releases/tag/v1.0.0-beta.1

Please feel free to try it out and let us know if you're facing issues.
Thanks!
